remote-trait-object-tests/src/test_store/man.rs

```diff
@@ -130,3 +130,34 @@ fn test_order3() {
     }
     test_runner(f);
 }
+
+#[test]
+fn drop_service_which_holds_remote() {
```

Comment:
What is the important part of this test?
Is it the last line `drop(store);`? If it is right, how about adding a comment like "drop shot nod fail"?